README.md

```diff
@@ -23,6 +23,33 @@ git clone https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git
 cd v2x-ros-conversion/docker
 ./build-image.sh
 ```
+
+To incorporate the v2x-ros-conversion package into a docker project, call the following command to clone the v2x-ros-conversion package into the project:
+```
+git clone --depth=1 https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git ${dir}/src/v2x-ros-conversion --branch "develop"
+```
+Once installed, the required dependencies for the v2x-ros-conversion package need to be installed prior to building the package. To do so, run 
+```
+sudo ~/src/v2x-ros-conversion/docker/install_dependencies.sh
```

Comment:
Same notes as #17 we dont need to run this script when building docker image - this will be called during the build process.